codegen-winter/README.md

```diff
@@ -0,0 +1,44 @@
+# Winterfell Code Generator
+
+This crate contains a code generator targeting the [Winterfell prover](https://github.com/novifinancial/winterfell) Rust library.
+
+The purpose of this code generator is to convert a provided `AirIR` representation of an AIR into a custom Rust struct that implements Winterfell's `Air` trait. The generated code can be used instead of writing a custom Winterfell `Air` implementation directly in Rust.
+
+## Generating the Winterfell Rust Code
+
+Generate Rust code from an `AirIR` (AirScript's intermediate representation) by instantiating a new `CodeGenerator` with an AirScript AST (the output of the AirScript parser) and then calling `generate`. `generate` will return the Rust code implementation as a `String`.
```

Comment:
Minor nit: I'm wondering if the start of sentence can be with `generate` or if we should change it to something like "The `generate` method will return".